lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -11,43 +11,44 @@ class StripePaymentIntentsGateway < StripeGateway
       CONFIRM_INTENT_ATTRIBUTES = %i[receipt_email return_url save_payment_method setup_future_usage off_session]
       UPDATE_INTENT_ATTRIBUTES = %i[description statement_descriptor_suffix statement_descriptor receipt_email setup_future_usage]
       DEFAULT_API_VERSION = '2020-08-27'
-      NO_WALLET_SUPPORT = %w(apple_pay google_pay android_pay)
 
       def create_intent(money, payment_method, options = {})
-        card_source_pay = payment_method.source.to_s if defined?(payment_method.source)
-        card_brand_pay = card_brand(payment_method) unless payment_method.is_a?(String) || payment_method.nil?
-        if NO_WALLET_SUPPORT.include?(card_source_pay) || NO_WALLET_SUPPORT.include?(card_brand_pay)
-          store_apple_or_google_pay_token = 'Direct Apple Pay and Google Pay transactions are not supported. Those payment methods must be stored before use.'
-          return Response.new(false, store_apple_or_google_pay_token)
-        end
-        post = {}
-        add_amount(post, money, options, true)
-        add_capture_method(post, options)
-        add_confirmation_method(post, options)
-        add_customer(post, options)
-        result = add_payment_method_token(post, payment_method, options)
-        return result if result.is_a?(ActiveMerchant::Billing::Response)
-
-        add_external_three_d_secure_auth_data(post, options)
-        add_metadata(post, options)
-        add_return_url(post, options)
-        add_connected_account(post, options)
-        add_radar_data(post, options)
-        add_shipping_address(post, options)
-        setup_future_usage(post, options)
-        add_exemption(post, options)
-        add_stored_credentials(post, options)
-        add_ntid(post, options)
-        add_claim_without_transaction_id(post, options)
-        add_error_on_requires_action(post, options)
-        add_fulfillment_date(post, options)
-        request_three_d_secure(post, options)
-
-        CREATE_INTENT_ATTRIBUTES.each do |attribute|
-          add_whitelisted_attribute(post, options, attribute)
+        MultiResponse.run do |r|
```

Comment:
I'm trying to figure out if changing from a single `Response` to a `MultiResponse` for existing calls could have side effects for library users.  We could instead check for `NetworkTokenizationCreditCard` before determining which type of response we create, but that means two different types could be returned from the same call.

Reply:
Running our system tests using this branch didn't detect any issues; I think `MultiResponse` is smart enough to handle this change gracefully.